Shared/Coordinators/SettingsCoordinator.swift

```diff
@@ -71,6 +72,10 @@ final class SettingsCoordinator: NavigationCoordinatable {
             let viewModel = QuickConnectSettingsViewModel()
             QuickConnectSettingsView(viewModel: viewModel)
         }
+
+        func makeFontPicker(onFontPick: @escaping (UIFontDescriptor) -> Void) -> NavigationViewCoordinator<FontPickerCoordinator> {
```

Comment:
Since no coordinator is needed, can just be:
```
        func makeFontPicker() -> some View {
            FontPickerView()
                .navigationTitle(L10n.subtitleFont)
        }
```